opm/parser/eclipse/EclipseState/Schedule/Group/GuideRate.hpp

```diff
@@ -75,11 +79,15 @@ struct GuideRateValue {
         return !(*this == other);
     }
 
-    double sim_time;
-    double value;
-    GuideRateModel::Target target;
+    double sim_time { std::numeric_limits<double>::lowest() };
+    double value { std::numeric_limits<double>::lowest() };
+    GuideRateModel::Target target { GuideRateModel::Target::NONE };
 };
 
+struct GRValAry {
```

Comment:
As I understand it this is a pair of current and previous value - then a name which alludes to array(?) might be a bit misleading?

Reply:
This structure started out as wrapper around `std::array` and the naming reflected that.  I'll think about a better name.